toys/slot-machine/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "slot-machine"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+sbor = { path = "../../../sbor" }
+scrypto = { path = "../../../scrypto" }
+
+[dev-dependencies]
+radix-engine = { path = "../../../radix-engine" }
```

Instead of relative paths, please use the git link: 
```suggestion
sbor = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "v0.3.0" }
scrypto = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "v0.3.0" }

[dev-dependencies]
radix-engine = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "v0.3.0" }
```